plugins/outputs/yandex_cloud_monitoring/README.md

```diff
@@ -0,0 +1,29 @@
+# Yandex Cloud Monitoring
+
+This plugin will send custom metrics to Yandex Cloud Monitoring. 
+https://cloud.yandex.com/services/monitoring
+
+### Configuration:
+
+```toml
+[[outputs.yandex_cloud_monitoring]]
+  ## Timeout for HTTP writes.
+  # timeout = "20s"
+
+  ## Normally should not be changed
+  # endpoint_url = "https://monitoring.api.cloud.yandex.net/monitoring/v2/data/write"
+
+  ## Normally folder ID is taken from Compute instance metadata
+  # folder_id = "..."
+
+  ## Can be set explicitly for authentification debugging purposes 
+  # iam_token = "..."  
+```
```

Comment:
missing some supported fields here.
commented values should be set to whatever the default values are.

Reply:
Done